versions/latest/modules/en/pages/reference/resource_profiling.adoc

```diff
@@ -26,7 +26,7 @@ The resource tests were intended to address the following problem statements:
 
 The tested components are:
 
-* RKE2 v1.27.12 with all packaged components enabled, Canal as the CNI
+* RKE2 v1.27.12 with all packaged components enabled, canal as the CNI
```

Comment:
Canal is a proper noun so this change doesn't seem correct unless it's referring to a code/actual value in which it should be enclosed by backticks.

```suggestion
* RKE2 v1.27.12 with all packaged components enabled, Canal as the CNI
```